src/App.tsx

```diff
@@ -1,10 +1,15 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import './App.scss';
-import { HomePage } from './modules/HomePage';
+import { BurgerMenu } from './components/BurgerMenu';
 
-const App = () => (
-  <div className="App">
-    <HomePage />
-  </div>
-);
+const App = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<BurgerMenu />} />
+      </Routes>
+    </BrowserRouter>
+  );
+};
```

Comment:
remove this because it should do Ilia. Also, we don't need a burger menu in App.